feature-tests/tests/resources/features/vcsGET/InvokeApiGateWay-HappyPath.feature

```diff
@@ -0,0 +1,9 @@
+Feature: Invoke-APIGateway-HappyPath.feature
+
+    Scenario Outline: Happy Path - Get Request to /ais/userId - Returns Data for a userId
+        Given I send a request to sqs queue with <aisType> data
+        When I invoke apiGateway to retreive the status of the userId with <historyValue>
```

Comment:
```suggestion
        When I invoke apiGateway to retreive the status of the userId without <historyValue>
```
Not sure about how to word this, but since you've set historyValue to false the history won't be included in the api response.

Reply:
Thanks Karla, we will update these steps to be a bit clearer on a future PR when we add the Endpoint functionality alongside the SQS functionality currently used.